src/platform/windows.rs

```diff
@@ -166,6 +170,27 @@ impl WindowExtWindows for Window {
     self.window.set_skip_taskbar(skip);
   }
 
+  #[inline]
+  fn set_progress_bar(&self, progress: ProgressBarState) {
+    if let Some(state) = progress.state {
+      let taskbar_state = {
+        match state {
+          TaskbarProgressState::None => 0,
+          TaskbarProgressState::Intermediate => 1,
+          TaskbarProgressState::Normal => 2,
+          TaskbarProgressState::Paused => 4,
+        }
+      };
+
+      self.window.set_taskbar_progress_state(taskbar_state);
+    }
+    if let Some(value) = progress.progress {
+      let progress = if value > 100 { 100f64 } else { value };
+
+      self.window.set_taskbar_progress(progress);
+    }
+  }
```

Comment:
same as above, move this implementation to `src/platform_impl/windows/window.rs`